### ISSUE TITLE
Design system font clean-up

### ISSUE THREAD

**Issue by lukefretwell** [cagov/Alpha#73]
- [x] Make sure 'Public Sans' is the default font for all styles
- [x] Any font designated at '300' should be '400' (@artuoma I think this is what we should do - on the Cal grant prototype, the .lead class looks thin)

**Comment by lukefretwell** [comment]
@abquirarte @KimberlyglennTSM added this new issue to sprint because it relates to #21 and we're just breaking it out.

**Comment by abquirarte** [comment]
@artuoma can you link to where this is now done?

**Comment by artuoma** [comment]
@abquirarte Sure sorry for the delay! The change is made in both the UX and CWDS repos:

- [ ] https://cagov.github.io/UX/cal-grant/

- [ ] https://cwds.dev/docs/4.3/content/typography/ 

- [ ] Completed

**Comment by abquirarte** [comment]
✅ thank you @artuoma ! 👍
